### PR TITLE
Fix summary panel UI bugs and reviews and setCategories bug

### LINKED ISSUE #110: Reviews get deleted whenever the SetCategories command is called

### PR DESCRIPTION
**Pull request by weixin-koh**
- Closes #110
- Fixes disappearing placeholder for summary panel bug
- Restructure the display of restaurant summary

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -188,28 +194,26 @@
...
     -fx-font-size: 20px;
-    -fx-padding: 5 0 10 0;
-}
-
-.vbox_label_header.label {
-    -fx-font-family: "Segoe UI";
-    -fx-font-weight: bold;
-    -fx-text-fill: white;
-    -fx-font-size: 17px;
+    -fx-padding: 5 0 0 0;
```
Comment (rbth7e5) [inline src/main/resources/view/DarkTheme.css]: use padding top

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -188,28 +194,26 @@
...
+    -fx-text-alignment: center;
+    -fx-wrap-text: true;
 }
 
 .vbox_label {
     -fx-font-family: "Segoe UI";
     -fx-text-fill: white;
     -fx-font-size: 15px;
-    -fx-padding: 0 0 5 0;
+    -fx-padding: 0 0 10 0;
```
Comment (rbth7e5) [inline src/main/resources/view/DarkTheme.css]: use padding bottom

**Review by rbth7e5 — CHANGES_REQUESTED** [review]
Fix the css issues. Otherwise it is good! Thanks for fixing the setCategories bug by the way!

File: src/main/resources/view/DarkTheme.css [hunk 3]
```diff
@@ -176,14 +176,15 @@
 
 .vbox_background {
     -fx-background-color: #3e7b91;
-    -fx-padding: 5 5 5 5;
+    -fx-padding: 10 10 10 10;
```
Comment (rbth7e5) [inline src/main/resources/view/DarkTheme.css:179]: if everything is 10 just put a single number 10. lel

Comment (weixin-koh) [reply src/main/resources/view/DarkTheme.css:179]: Omg I didn't know that you could do this, thanks for the tip!

**Review by rbth7e5 — APPROVED** [review]

**Review by Wklee96 — APPROVED** [review]

**Pull request merged**